### ISSUE TITLE
Launcher fails to fetch `winstore_versions.json`

### ISSUE THREAD

**Issue by remireyes** [BedrockLauncher/BedrockLauncher#269]
Days ago I opened an issue with the same problem, but it seems I had an older version of the launcher, now, with the lastest version avaible, it still happens.

LOGS:
```
2022-08-07 21:03:26.1705|DEBUG||Version: 2022.7.4.195
2022-08-07 21:03:26.2738|DEBUG||Git Repo: https://github.com/BedrockLauncher/BedrockLauncher
2022-08-07 21:03:26.2738|DEBUG||Git Branch: lite
2022-08-07 21:03:26.2738|DEBUG||Git Commit: 3ed563d
2022-08-07 21:03:26.2738|DEBUG||Git Sha: 3ed563d2826badcfafa0817d0df8700c778f1e42
2022-08-07 21:03:26.2738|DEBUG||Application Starting...
2022-08-07 21:03:26.7635|DEBUG||Application Initalization Started!
2022-08-07 21:03:26.7699|DEBUG||Application Initalization Finished!
2022-08-07 21:03:28.9271|DEBUG||Preparing Application...
2022-08-07 21:03:32.7359|DEBUG||LoadTextDBVersions Failed! Generating Blank VersionTextDb
2022-08-07 21:03:32.7359|DEBUG||File: C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\winstore_technical_versions.txt
2022-08-07 21:03:32.7863|DEBUG||System.IO.FileNotFoundException: Could not find file 'C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\winstore_technical_versions.txt'.
File name: 'C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\winstore_technical_versions.txt'
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.FileStreamHelpers.ChooseStrategy(FileStream fileStream, String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
   at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
   at System.IO.File.OpenText(String path)
   at BedrockLauncher.UpdateProcessor.Databases.VersionTextDb.ReadFile(String filePath) in D:\a\BedrockLauncher\BedrockLauncher\modules\BedrockLauncher.UpdateProcessor\Databases\VersionTextDb.cs:line 57
   at BedrockLauncher.UpdateProcessor.Handlers.VersionManager.LoadTextDBVersions(String filePath) in D:\a\BedrockLauncher\BedrockLauncher\modules\BedrockLauncher.UpdateProcessor\Handlers\VersionManager.cs:line 205
2022-08-07 21:03:32.7863|DEBUG||LoadJsonDBVersions Failed! Generating Blank VersionJsonDb
2022-08-07 21:03:32.7863|DEBUG||File: C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\winstore_versions.json
2022-08-07 21:03:32.7863|DEBUG||System.IO.FileNotFoundException: Could not find file 'C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\winstore_versions.json'.
File name: 'C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\winstore_versions.json'
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.FileStreamHelpers.ChooseStrategy(FileStream fileStream, String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
   at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
   at System.IO.File.OpenText(String path)
   at BedrockLauncher.UpdateProcessor.Databases.VersionJsonDb.ReadJson(String filePath, Dictionary`2 architectures) in D:\a\BedrockLauncher\BedrockLauncher\modules\BedrockLauncher.UpdateProcessor\Databases\VersionJsonDb.cs:line 34
   at BedrockLauncher.UpdateProcessor.Handlers.VersionManager.LoadJsonDBVersions(String filePath) in D:\a\BedrockLauncher\BedrockLauncher\modules\BedrockLauncher.UpdateProcessor\Handlers\VersionManager.cs:line 186
2022-08-07 21:03:33.7179|DEBUG||Checking for updates
2022-08-07 21:03:35.8836|DEBUG||Current tag: 2022.7.4.195
2022-08-07 21:03:35.8836|DEBUG||Latest tag: 2022.7.4.195
2022-08-07 21:03:35.8836|DEBUG||Preparing Application: DONE
2022-08-07 21:03:40.6736|DEBUG||Skipping package removal - same path: Microsoft.MinecraftUWP_1.19.5.0_x64__8wekyb3d8bbwe C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\versions\Minecraft-28d2f975-a0fc-401a-9987-9a6a117f8922
2022-08-07 21:03:40.6875|DEBUG||Registering package
2022-08-07 21:03:42.1949|DEBUG||Deployment done: Completed
2022-08-07 21:03:42.1949|DEBUG||App re-register done!
2022-08-07 21:03:58.4120|DEBUG||App launch finished!
```

**Comment by remi-reyes** [comment]
What are these files for? Can I fix it?

**Comment by Rayth** [comment]
> What are these files for? Can I fix it?

Tbh I'm not sure. I checked my files, and I have those files however they don't contain the latest versions. They contain versions from when I last used the official launcher at the start of july

**Comment by remi-reyes** [comment]
I also checked the folder for them, but theres none, I could try to just write blank documents

**Comment by Rayth** [comment]
I'm assuming you left the launcher to read your MC data from the default location the first time you ran it, whereas I changed my location. I have updated files in the location I chose. What you could do is go to C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\ on your PC in windows explorer, and add these 2 files: This was updated today so contains all the latest versions.
[winstore_versions.json.txt](https://github.com/BedrockLauncher/BedrockLauncher/files/9305123/winstore_versions.json.txt)
[winstore_technical_versions.txt](https://github.com/BedrockLauncher/BedrockLauncher/files/9305128/winstore_technical_versions.txt)
 
 Note: the one called winstore_versions.json.txt, change it back to a json file by removing the .txt from the end of the name (make sure windows shows file extensions)

**Comment by remi-reyes** [comment]
I did it, now the error doesn't appear anymore, however, the game still crashes a few moments after launching.

these are the new logs:
`
2022-08-10 18:09:18.9160|DEBUG||Version: 2022.7.4.195
2022-08-10 18:09:19.0636|DEBUG||Git Repo: https://github.com/BedrockLauncher/BedrockLauncher
2022-08-10 18:09:19.0636|DEBUG||Git Branch: lite
2022-08-10 18:09:19.0636|DEBUG||Git Commit: 3ed563d
2022-08-10 18:09:19.0636|DEBUG||Git Sha: 3ed563d2826badcfafa0817d0df8700c778f1e42
2022-08-10 18:09:19.0781|DEBUG||Application Starting...
2022-08-10 18:09:19.7284|DEBUG||Application Initalization Started!
2022-08-10 18:09:19.7284|DEBUG||Application Initalization Finished!
2022-08-10 18:09:21.9727|DEBUG||Preparing Application...
2022-08-10 18:09:26.6299|DEBUG||Checking for updates
2022-08-10 18:09:27.6703|DEBUG||Current tag: 2022.7.4.195
2022-08-10 18:09:27.6703|DEBUG||Latest tag: 2022.7.4.195
2022-08-10 18:09:27.6703|DEBUG||Preparing Application: DONE
2022-08-10 18:09:59.4239|DEBUG||Skipping package removal - same path: Microsoft.MinecraftUWP_1.19.5.0_x64__8wekyb3d8bbwe C:\Users\José Reyes\AppData\Roaming\.minecraft_bedrock\versions\Minecraft-28d2f975-a0fc-401a-9987-9a6a117f8922
2022-08-10 18:09:59.4368|DEBUG||Registering package
2022-08-10 18:10:01.1073|DEBUG||Deployment done: Completed
2022-08-10 18:10:01.1073|DEBUG||App re-register done!
`

**Comment by Rayth** [comment]
That doesn't look like an issue with the launcher anymore.

**Comment by Rayth** [comment]
Just a thought, since the launcher isn't throwing any bugs now, can you open minecraft from your start menu without opening the launcher? I personally only really use the launcher to switch which version is installed on my device but otherwise open MC normally from start

**Comment by remi-reyes** [comment]
> Just a thought, since the launcher isn't throwing any bugs now, can you open minecraft from your start menu without opening the launcher? I personally only really use the launcher to switch which version is installed on my device but otherwise open MC normally from start

When I do it still crashes, I don't know if the issue has to do with something like the way the game gets installed

**Comment by Rayth** [comment]
Can anyone confirm if they're still getting this error on the new version?

**Comment by Rayth** [comment]
This is fixed in 2023.03.12.2
